language/entity-extraction/src/main/java/com/google/cloud/language/automl/entity/extraction/samples/CreateDataset.java

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
The file path for this seems really long. 
`java-docs-samples/language/entity-extraction/src/main/java/com/google/cloud/language/automl/entity/extraction/samples/`

Could it be: `java-docs-samples/language/entity-extraction/src/main/java/com/example/entityextraction/` or something along those lines?

Reply:
Will change it to ```entity-extraction``` and ```sentiment-analysis``` in the latest commit.

Reply:
Not sure if this is intended or a maven thing, but there still seems to be a bunch of in-between folders that seem unnecessary:
`java-docs-samples/language/entity-extraction/src/main/java/com/google/cloud/language/automl/entityextraction/samples/`
Which gives us this package name in the source files: 
`package com.google.cloud.language.automl.entityextraction.samples;`

Where our standard is:
`java-docs-samples/language/cloud-client/src/main/java/com/example/language/`
Which gives us this package name: 
`package com.example.language;`

Seems like part of the folder structure above: `com/google/cloud/language/automl/entityextraction/samples/`
Could be cut to: `com/example/entityextraction`
To give us this package name: 
`package com.com/example/entityextractions;`